.github/workflows/build.yml

```diff
@@ -28,9 +28,23 @@ jobs:
       - name: Test
         run: go test -v -cover -timeout=30s ./...
 
-  nightly:
-    uses: charmbracelet/meta/.github/workflows/nightly.yml@main
-    secrets:
-      docker_username: ${{ secrets.DOCKERHUB_USERNAME }}
-      docker_token: ${{ secrets.DOCKERHUB_TOKEN }}
-      goreleaser_key: ${{ secrets.GORELEASER_KEY }}
+  goreleaser:
+    env:
+      GO111MODULE: "on"
+      DOCKER_CLI_EXPERIMENTAL: enabled
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "~1.17"
+      - uses: actions/checkout@v2
+      - uses: anchore/sbom-action/download-syft@v0.6.0
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: goreleaser/goreleaser-action@v2
+        with:
+          distribution: goreleaser-pro
+          version: latest
+          args: --snapshot --skip-sign
```

Comment:
should probably add `--skip-sbom` as well

also, maybe move this to the meta repo and just use it here? likely to be common on several other projects as well